submit_slurm_cpu.sh

```diff
@@ -0,0 +1,28 @@
+JOB_NAME="fbopt"
+VENV="domainlab_py39"
+BASHRC="~/.bashrc"  # source ~/.bash_profile
+PATH_CODE=$1
+PATH_OUT_BASE="${PATH_CODE}/job_logs"
+PATH_YAML=$2
+START_SEED=$3
+
+
+job_file="${PATH_OUT_BASE}/${JOB_NAME}.cmd"
+echo "#!/bin/bash
+#SBATCH -J ${JOB_NAME}
+#SBATCH -o ${PATH_OUT_BASE}/${JOB_NAME}.out
+#SBATCH -e ${PATH_OUT_BASE}/${JOB_NAME}.err
+#SBATCH -p cpu_p 
+#SBATCH -t 2-00:00:00
```

Comment:
@drEast, how to make this time longer? can we make it 4 days or longer?  if so, what should we put here?